server/src/types/types.ts

```diff
@@ -1,38 +1,38 @@
 export interface Script {
-  name: string
-  imageSrc: string
-  characters: Character[]
+  name: string;
+  imageSrc: string;
+  characters: Character[];
 }
 export interface UnifiedGame {
-  playersToNames: Record<string, string>
-  playersToRoles: PlayerSet
-  gmSecretHash: string
-  gameStarted: boolean
+  playersToNames: Record<string, string>;
+  playersToRoles: PlayerSet;
+  gmSecretHash: string;
+  gameStarted: boolean;
 }
 
-export type PlayerSet = Record<string, string>
+export type PlayerSet = Record<string, string>;
 
 export interface Self {
-  name: string | undefined
-  role: string | undefined
+  name: string | undefined;
+  role: string | undefined;
 }
 
 export interface Character {
-  id: string
-  name: string
-  team: string
-  imageSrc?: string
+  id: string;
+  name?: string;
```

Comment:
Making `name` optional is the only real change here.

Reply:
you don't have eslint set up correctly